sections/design/prototypes.tex

```diff
@@ -0,0 +1,77 @@
+\section{Prototypes}
+To get a clear view of how the different pages will look and catch any early potential design problems, we have created prototypes for each of the different pages. 
+We will use these to make the implementation of the different pages easier make ensure that we have a consistent design througout the system. 
+The prototypes will not serve as an exact vision of how the pages will look but will instead serve as a guideline to how the page should be set up.
+In this section we will look at some of the more interesting prototypes where we have had to make some design choices.
+\begin{figure}[H]
+    \includegraphics[width=\linewidth]{/prototypes/admin-dashboard.png}
+     \caption{Shows the admin dashboard for admins that are logged into the system}
+     \label{fig:admin-dashboard}
+ \end{figure}
+The admin dashboard gives the admin an overview of the recently reported tutors and the different categories. 
+The admin needs to see tutor reports since they are the once that will handle them and decide if they should be banned. 
+Admins are also in charge of the different teaching categories and therefore they need to have an overview of the current ones and also be able to add new.
+We also have a top bar that is used on every page for when the user is logged in. It shows a couple of different icons where the user can go to the inbox, the calendar or the user's own page. It will also show notifications for when different events related to the user happens in the system. On the far left side of the top bar we consider having an icon to go to the landing page but this have not been decided yet
```

Comment:
```suggestion
We also have a top bar that is used on every page for when the user is logged in. It shows a couple of different icons where the user can go to the inbox, the calendar or the user's own page. 
It will also show notifications for when different events related to the user happen in the system. 
On the far left side of the top bar we considered having an icon to go to the landing page but this has not been decided yet
```

Reply:
ift sidste sætning - hvad så når det bliver decided, skal vi opdatere? tror vi skal tage en executive decision før vi merger det her ind og så enten slette eller skrive at den er der